core/store/src/config.rs

```diff
@@ -174,6 +189,31 @@ impl StoreConfig {
             _ => bytesize::ByteSize::mib(32),
         }
     }
+
+    fn default_per_shard_max_bytes() -> HashMap<ShardUId, bytesize::ByteSize> {
+        let epoch_config_store = EpochConfigStore::for_chain_id(MAINNET, None).unwrap();
+        let mut shard_layouts: Vec<ShardLayout> = Vec::new();
+        // 72 is the latest protocol version of config store as of writing
+        for protocol_version in 72..=PROTOCOL_VERSION {
```

Comment:
Hardcoding 72 is not the pretties but it certainly does the job. Nice hack! 

nit: Can you add a comment saying that it should actually look for shard layouts starting from the current chain head but it's not available here? 

nit: Instead of 72 can you use ProtocolFeature::SimpleNightshadeV4::protocol_version()?

Reply:
Actually based on @shreyan-gupta's comment below we'd probably want `SimpleNightshadeV4 - 1`

Reply:
> Can you add a comment saying that it should actually look for shard layouts starting from the current chain head but it's not available here?

Sure thing!

> Instead of 72 can you use ProtocolFeature::SimpleNightshadeV4::protocol_version()?

Sure. I chose 72 coz it's the latest version specified in the config store that was live on mainnet so it should guaranteed to work. I guess `SimpleNightshadeV4` is gonna be stablized soon?

Reply:
Now that I think of it, how about `PROTOCOL_VERSION - 1`?